server/views/admin/users/index.njk

```diff
@@ -72,13 +72,13 @@
           <div class="govuk-grid-column-one-quarter">
             {{ govukSelect({
                 label: {
-                  text: "Regions",
+                  text: "AP Areas",
                   classes: "govuk-label--s"
                 },
                 classes: "govuk-input--width-20",
-                id: "region",
-                name: "region",
-                items: convertObjectsToSelectOptions(regions, 'All regions', 'name', 'id', 'selectedRegion')
+                id: "areas",
+                name: "areas",
```

Comment:
Should this be `area`?

Reply:
I went for aligning this with the other drop down ids rather than the previous id - which use a plural in the name e.g. `qualifications`, and also the label is in plural. But in general singular does probably make more sense